packages/comms/src/ecl/dfuWorkunit.ts

```diff
@@ -0,0 +1,332 @@
+import { Cache, IEvent, scopedLogger, StateCallback, StateEvents, StateObject, StatePropCallback } from "@hpcc-js/util";
+import { IConnection, IOptions } from "../connection";
+import { ESPExceptions } from "../espConnection";
+import { WsSMC } from "../services/wsSMC";
+import { FileSpray, SprayFixedEx, FileSprayService, SprayVariableEx } from "../services/fileSpray";
+import * as WsTopology from "../services/wsTopology";
+
+const logger = scopedLogger("dfuWorkunit.ts");
```

Comment:
Something like:  @hpcc-js/comms/ecl/dfuWorkunits.ts would be preferable (I know we have been inconsistent with this)